src/Maestro/SubscriptionActorService/Program.cs

```diff
@@ -35,6 +37,7 @@ private static void Main()
 
     public static void Configure(IServiceCollection services)
     {
+        services.TryAddTransient<ILogger>(sp => sp.GetRequiredService<ILogger<SubscriptionActor>>());
```

Comment:
Because the code in this project passes `ILogger` around as paramaters instead of proper DI but DI can only create `ILogger<T>` so classes like `CoherencyResolver` that require to be used in both scenarios cannot have 2 ctors with `ILogger` and `ILogger<T>` as DI doesn't know which one to use.
So I had keep `ILogger` in the ctor but that means I need to teach DI to resolve that.

Reply:
Ahh I see, thanks